docker/deploy-fedimintd/docker-compose.yaml

```diff
@@ -58,8 +58,9 @@ services:
       - FM_BITCOIN_NETWORK=bitcoin
       - FM_BIND_P2P=0.0.0.0:8173
       - FM_P2P_URL=fedimint://${FM_DOMAIN}:8173
-      - FM_BIND_API=0.0.0.0:8174
       - FM_API_URL=wss://${FM_DOMAIN}/ws/
+      - FM_BIND_API_WS=172.20.0.11:8174
```

Comment:
On the standard docker, I think we only need to expose the http/ws api and ui on internal docker network, so it's not accidentally exposed in plaintext outside.

Reply:
Did you have a chance to test the deploy to the hetzner envs?

Reply:
Nope. :D